articles/spring-apps/how-to-enterprise-service-registry.md

```diff
@@ -18,7 +18,7 @@ ms.custom: devx-track-java, event-tier1-build-2022
 
 This article shows you how to use VMware Tanzu® Service Registry with Azure Spring Apps Enterprise tier.
 
-The [Tanzu Service Registry](https://docs.vmware.com/en/Spring-Cloud-Services-for-VMware-Tanzu/2.1/spring-cloud-services/GUID-service-registry-index.html) is one of the commercial VMware Tanzu components. This component helps you apply the *service discovery* design pattern to your applications.
+The [Tanzu Service Registry](https://docs.vmware.com/en/Spring-Cloud-Services-for-VMware-Tanzu) is one of the commercial VMware Tanzu components. This component helps you apply the *service discovery* design pattern to your applications.
```

Comment:
should it be scoped with version in url as well?

Reply:
Service registry is a special version from VMware. We don't have its public version number.

Reply:
@dingmeng-xue the link target doesn't mention Tanzu Service Registry. We generally want the visible link text to match the H1 heading of the link target, or at least be similar enough to avoid reader confusion. Can we move the link to an additional sentence at the end of this paragraph? Searching from that location for "Tanzu Service Registry" gives a result with the URL you are replacing, but requiring the reader to search may be a better alternative at this time, for when a new link becomes available.  

```suggestion
The Tanzu Service Registry is one of the commercial VMware Tanzu components. This component helps you apply the *service discovery* design pattern to your applications. For more information, see [Spring Cloud Services for VMware Tanzu](https://docs.vmware.com/en/Spring-Cloud-Services-for-VMware-Tanzu).
```

Reply:
@dingmeng-xue @KarlErickson as there is no direct mapping public url for Tanzu service registry, then I'd suggest removing the hyperlink here. And later after VMware published its related doc, we can add it. and to Karl, the hyperlink's purpose is to let customers be aware of the corresponding Tanzu components we integrate with but no intention to let them learn more from there, so we should avoid making this kind of link too explicit in docs here.

Reply:
I have removed the link based on what we have discussed.

Reply:
I have removed the link based on what we have discussed.